taxi/views.py

```diff
@@ -79,6 +83,24 @@
     success_url = reverse_lazy("taxi:car-list")
 
 
+class CarDriverUpdateView(LoginRequiredMixin, generic.View):
+    @staticmethod
+    def post(
+        request: HttpRequest, pk: int
+    ) -> HttpResponseRedirect | HttpResponseBadRequest:
+        try:
+            car = Car.objects.get(pk=pk)
+        except Car.DoesNotExist:
+            return HttpResponseBadRequest("Car does not exist")
+        drivers = car.drivers.all
+
+        if request.user in drivers:
+            drivers.remove(request.user)
+        else:
+            drivers.add(request.user)
```

Comment:
The logic here assumes that `drivers` is a ManyToManyField. Ensure that the `drivers` field in the `Car` model is indeed a ManyToManyField to support adding and removing users.